lib/drivers/medtronic600/cli/blob_loader.js

```diff
@@ -49,7 +49,7 @@ function loadFile(filePath, tz, userid) {
         filename: filePath,
         fileData: json,
         timezone: tz,
-        version: `${pkg.name} ${pkg.version}`,
```

Comment:
Is this what caused the issue?

Reply:
Yes, but the `platform` was returning back an error message that was invalid JSON. @darinkrauss has added it to his TODO list.